#!/usr/bin/env python
PACKAGE = "vision_buoy"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("red_min_param",    int_t,    0, "red_min parameter", 0,  0, 260)
gen.add("red_max_param",    int_t,    0, "red_max parameter", 0,  0, 260)
gen.add("green_min_param",    int_t,    0, "green_min parameter", 0,  0, 260)
gen.add("green_max_param",    int_t,    0, "green_max parameter", 0,  0, 260)
gen.add("blue_min_param",    int_t,    0, "green_min parameter", 100,  0, 260)
gen.add("blue_max_param",    int_t,    0, "green_max parameter", 255,  0, 260)

exit(gen.generate(PACKAGE, "vision_buoy", "buoy"))
